src/els_dap_general_provider.erl

```diff
@@ -64,6 +64,11 @@ init() ->
 
 -spec handle_request(request(), state()) -> {result(), state()}.
 handle_request({<<"initialize">>, _Params}, State) ->
+  %% quick fix to satisfy els_config initialization
+  {ok, RootPath} = file:get_cwd(),
+  RootUri = unicode:characters_to_binary(["file://", RootPath]),
```

Comment:
You can use `els_uri:uri(els_utils:to_binary(Cwd))` to ensure compatibility with Windows.